.gitignore

```diff
@@ -1,2 +1,4 @@
 /.idea/
-package-lock.json
+/node_modules
+/package-lock.json
```

Comment:
lock files should be commited.

Reply:
I've [asked](https://github.com/stac-extensions/template/issues/8) why they excluded it.

Reply:
Looks like their reasons are different from ours, so I'll just add it.